package.json

```diff
@@ -12,8 +12,7 @@
   "types": "./index.d.ts",
   "exports": {
     "require": "./index.js",
-    "import": "./index.mjs",
-    "types": "./index.d.ts"
```

Comment:
remove this this line will make ts developer with tsconfig `"moduleResolution": "bundler"` can't import this package

Reply:
No, it doesn’t. This line states that instead of using `index.d.mts`, TypeScript must always resolve to the CJS types in `index.d.ts`. Removal of this field is a crucial to make it work with the `node16` / `nodenext` / `bundler` module resolutions.

Why do you think this breaks it?

Reply:
Yes, you are right, my mistake, this works.